circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  timezone:
+    America/Los_Angeles
+
+test:
+  override:
+    - echo "No tests are needed"
```

Comment:
:( Given that this is such a small project right now, and it's so important that things not be broken for events, I think it's pretty important that we have tests for this. Otherwise if somebody commits something that breaks something and they don't see it, it won't get caught until it breaks during an event, and then it's already too late. @pjo256 you have experience writing karma tests. I think we should definitely have some unit tests for this making sure events show up when we expect them and in the right places.